renv.lock

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
Do you like `renv` better than using conda with R packages? I've always found conda impossible to use with R - interested in trying something different next time I have to write some R code.

Reply:
Yeah, it's great! (Or at least it's great so far, I haven't tried to load an renv environment on another machine so we'll see). On the download side it mostly just works like I want Conda to e.g. it automatically detects the difference between bioconductor and CRAN packages